src/stormpath.social-login.google.js

```diff
@@ -0,0 +1,61 @@
+(function() {
+  'use strict';
+
+  function GoogleLoginService($q, $spJsLoader) {
+    this.googleAuth = null;
+    this.$q = $q;
+    this.$spJsLoader = $spJsLoader;
+  }
+
+  angular.extend(GoogleLoginService.prototype, {
```

Comment:
Move the properties to the constructor and the functions as explicit GoogleLoginService.prototype declarations.